### ISSUE TITLE
Better parser for TAP output

### ISSUE THREAD

**Issue by rpunkfu** [rpunkfu/tap-that-html#2]

**Comment by gabrielcsapo** [comment]
Yeah getting good parser output was difficult, I ended up using tap-parser, but that when used against tape and tap tests differs greatly. Tape seems to be left behind while tap continues to move forward. Simple things like nesting tests and timing is not in Tape, while it is exists in Tap.
